target/linux/ramips/dts/VOCORE2.dtsi

```diff
@@ -0,0 +1,79 @@
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "vocore,vocore2", "mediatek,mt7628an-soc";
+
+	chosen {
+		bootargs = "console=ttyS2,115200";
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		wifi_led: wifi {
+		gpios = <&gpio1 12 GPIO_ACTIVE_HIGH>;
```

Comment:
Please move the gpio-leds node into the dts files. In the past I've done the same as you are doing here, but it has turned out to be not practical.